packages/one-app-bundler/webpack/loaders/validate-required-externals-loader.js

```diff
@@ -12,30 +12,47 @@
  * under the License.
  */
 
+const fs = require('node:fs');
+const path = require('node:path');
 const loaderUtils = require('loader-utils');
 const readPkgUp = require('read-pkg-up');
 
 function validateRequiredExternalsLoader(content) {
   const options = loaderUtils.getOptions(this);
   const { packageJson } = readPkgUp.sync();
+  const integrityManifest = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'bundle.integrity.manifest.json'), 'utf-8'));
+  const requiredExternals = options.requiredExternals.reduce((obj, externalName) => {
+    // eslint-disable-next-line global-require, import/no-dynamic-require -- need to require a package.json at runtime
+    const { version } = require(`${externalName}/package.json`);
+    const semanticRange = packageJson.dependencies[externalName];
 
-  const requiredExternals = options.requiredExternals.map((externalName) => {
-    const version = packageJson.dependencies[externalName];
-    return `'${externalName}': '${version}'`;
-  });
+    return {
+      ...obj,
+      [externalName]: {
```

Comment:
This change does mean that modules minimum version of one-app will be the version with externals release.